lib/btc/BtcRpc.js

```diff
@@ -45,15 +45,67 @@ class BtcRpc {
     return this.asyncCall('cmdlineUnlock', [time]);
   }
 
+  async sendMany({ batch, options }) {
+    for (let tx in batch) {
+      batch[tx] /= 1e8;
+    }
+    const array = ['', batch];
+    if (options) {
+      array.push(options);
+    }
+    return this.asyncCall('sendMany', array);
+  }
+
   async sendToAddress({ address, amount }) {
     return this.asyncCall('sendToAddress', [address, amount / 1e8]);
   }
 
+  async unlockAndSendManyBatched({ batchArray, passphrase, options, time = 10800 }) {
+    function RpcException(message) {
```

Comment:
pull this out

Reply:
So I throw a custom error object instead now copied here:

```
    if (failureBool) {
      throw {
        name: 'Batched Transaction Failure',
        message: 'At least one of many requests Failed',
        data: txFailureResults
      };
    }
```